docs/TheThingsMessage.md

```diff
@@ -45,17 +45,38 @@ You can also add other analog readings.
 Encode the message you want to send.
 
 ```c
-static void encodeSensorData(sensordata_t *data, byte **buffer, size_t *size);
+static void encodeSensorData(sensordata_t *data, byte **buffer, size_t bufSize, size_t *size);
```

Comment:
If you're already passing the size of the buffer to allocate, then let the user allocate the buffer and pass the buffer size in `*size`. I think it's nice if `encodeSensorData()` does the allocation, but then don't bother the user with the allocation, so use a `const` somewhere with the to-be-allocated bytes